src/test/java/millom/sandbox/Utility/JsonUtility.java

```diff
@@ -0,0 +1,208 @@
+package millom.sandbox.Utility;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonUtility {
+
+  public static final String SAMPLE_RESPONSE="{\n"
```

Comment:
Java has something called "text block", which makes writing lines like the following a bit easier.
See f.ex. https://docs.oracle.com/en/java/javase/15/text-blocks/index.html

But no need to change anything here though. Just for reference later on.

Reply:
changed **SAMPLE_RESPONSE, RESPONSE_MISSING_JSON_PROPERTY, RESPONSE_PROPERTY_MISMATCH** to **text block**.